src/database/sqlite/sqlite_aclk.c

```diff
@@ -338,6 +383,34 @@ static void aclk_synchronization(void *arg)
                 case ACLK_DATABASE_PUSH_ALERT:
                     aclk_push_alert_events_for_all_hosts();
                     break;
+
+                case ACLK_MQTT_WSS_CLIENT:
+                    config->client = (mqtt_wss_client) cmd.param[0];
+                    break;
+
+                case ACLK_QUERY_EXECUTE:;
```

Comment:
:;?

Reply:
just a black statement to bypass an issue in the IDE that would complain if the first statement is a variable definition. I will check if it has been fixed and removed -- it is used in other parts of the code as well.

Reply:
It gives a warning `Label followed by a declaration is a C23 extension` so the `;` handles that